Bogus_GMail:2.16.4.126.203.233.252.165.95.123.208.158.174.54.225.12.174.30.crt

```diff
@@ -0,0 +1,38 @@
+# alias="Bogus GMail"
+# trust=
+# distrust=CKA_TRUST_CODE_SIGNING CKA_TRUST_EMAIL_PROTECTION CKA_TRUST_SERVER_AUTH
```

Comment:
These should be removed (anything with stuff in the distrust field)

Reply:
ooooh, I misunderstood, I just looked at file names, will update